RecoHI/HiJetAlgos/python/RecoHiJets_EventContent_cff.py

```diff
@@ -1,45 +1,37 @@
 import FWCore.ParameterSet.Config as cms
 
-#Full Event content ---- temporary
+# AOD content
+RecoHiJetsAOD = cms.PSet(
+    outputCommands = cms.untracked.vstring(
+        'keep *_ak4CaloJets_*_*',
+        'keep *_akPu4CaloJets_*_*', 
+        'keep *_ak4PFJets_*_*',
+        'keep *_akPu3PFJets_*_*',
+        'keep *_akPu4PFJets_*_*',
+        'keep *_akCs3PFJets_*_*',
+        'keep *_akCs4PFJets_*_*',
+        'keep *_PFTowers_*_*',
+        'keep *_hiFJRhoProducer_*_*',
+        'keep CaloTowersSorted_towerMaker_*_*',
+        'keep recoPFCandidates_particleFlowTmp_*_*')
+)
+
+# RECO content
 RecoHiJetsRECO = cms.PSet(
-    outputCommands = cms.untracked.vstring( 'keep *_akPu3CaloJets_*_*',
-                                            'keep *_akPu4CaloJets_*_*', 
-                                            'keep *_akPu5CaloJets_*_*', 
-                                            'keep *_iterativeConePu5CaloJets_*_*', 
-                                            'keep *_akPu3PFJets_*_*',
-                                            'keep *_akPu4PFJets_*_*',
-                                            'keep *_akPu5PFJets_*_*',
-                                            'keep *_akCs3PFJets_*_*',
-                                            'keep *_akCs4PFJets_*_*',
-                                            'keep *_kt4PFJetsForRho_*_*',
-                                            'keep *_*HiGenJets_*_*',
-                                            'keep *_PFTowers_*_*',
-                                            'keep *_hiFJRhoProducer_*_*'
-
-                                           )
-    )
-
+    outputCommands = cms.untracked.vstring( 
+        'keep *_akPu3CaloJets_*_*', 
+        'keep *_akPu5CaloJets_*_*', 
+        'keep *_iterativeConePu5CaloJets_*_*', 
+        'keep *_akPu5PFJets_*_*',
+        'keep recoCandidatesOwned_caloTowers_*_*',
```

Comment:
This (or a similar) line was not present in the original event content

Reply:
In original event content, this caloTower not in AOD.

Based on the hi workflow (140.53/158.0) RECO output's event content list, the caloTowers exist.
`edm::OwnVector<reco::Candidate,edm::ClonePolicy<reco::Candidate> >    "caloTowers"                ""                "reRECO"`

So I keep this caloTower in RECO and FEVT in this file just in case.

But If you don't need this caloTower then I will remove this line.
@mandrenguyen :  could you please confirm this ?